Import from `store/constants` in published components is causing error

I encountered this error when trying to use `@woocommerce/components` in a new project:

```
ERROR in ./node_modules/@woocommerce/components/build-module/search/autocompleters/product-cat.js
    Module not found: Error: Can't resolve 'store/constants' in 'woocommerce-gutenberg-products-block/node_modules/@woocommerce/components/build-module/search/autocompleters'
```

Looks like we import the `NAMESPACE` constant, for publishing these we should just switch this to a local variable.

--- a/packages/components/src/search/autocompleters/coupons.js
+++ b/packages/components/src/search/autocompleters/coupons.js
@@ -13,7 +13,6 @@ import { stringifyQuery } from '@woocommerce/navigation';
  * Internal dependencies
  */
 import { computeSuggestionMatch } from './utils';
-import { NAMESPACE } from 'store/constants';
 
 /**
  * A coupon completer.
@@ -33,7 +32,7 @@ export default {
 			};
 			payload = stringifyQuery( query );
 		}
-		return apiFetch( { path: `${ NAMESPACE }coupons${ payload }` } );
+		return apiFetch( { path: `/wc/v3/coupons${ payload }` } );
 	},
 	isDebounced: true,
 	getOptionKeywords( coupon ) {

--- a/packages/components/src/search/autocompleters/product-cat.js
+++ b/packages/components/src/search/autocompleters/product-cat.js
@@ -13,7 +13,6 @@ import { stringifyQuery } from '@woocommerce/navigation';
  * Internal dependencies
  */
 import { computeSuggestionMatch } from './utils';
-import { NAMESPACE } from 'store/constants';
 
 /**
  * A product categories completer.
@@ -34,7 +33,7 @@ export default {
 			};
 			payload = stringifyQuery( query );
 		}
-		return apiFetch( { path: `${ NAMESPACE }products/categories${ payload }` } );
+		return apiFetch( { path: `/wc/v3/products/categories${ payload }` } );
 	},
 	isDebounced: true,
 	getOptionKeywords( cat ) {

--- a/packages/components/src/search/autocompleters/product.js
+++ b/packages/components/src/search/autocompleters/product.js
@@ -14,7 +14,6 @@ import { stringifyQuery } from '@woocommerce/navigation';
  */
 import { computeSuggestionMatch } from './utils';
 import ProductImage from '../../product-image';
-import { NAMESPACE } from 'store/constants';
 
 /**
  * A products completer.
@@ -35,7 +34,7 @@ export default {
 			};
 			payload = stringifyQuery( query );
 		}
-		return apiFetch( { path: `${ NAMESPACE }products${ payload }` } );
+		return apiFetch( { path: `/wc/v3/products${ payload }` } );
 	},
 	isDebounced: true,
 	getOptionKeywords( product ) {

--- a/packages/components/src/search/autocompleters/variations.js
+++ b/packages/components/src/search/autocompleters/variations.js
@@ -14,7 +14,6 @@ import { stringifyQuery, getQuery } from '@woocommerce/navigation';
  */
 import { computeSuggestionMatch } from './utils';
 import ProductImage from '../../product-image';
-import { NAMESPACE } from 'store/constants';
 
 /**
  * Create a variation name by concatenating each of the variation's
@@ -53,7 +52,7 @@ export default {
 		if ( ! product || product.includes( ',' ) ) {
 			console.warn( 'Invalid product id supplied to Variations autocompleter' );
 		}
-		return apiFetch( { path: `${ NAMESPACE }products/${ product }/variations${ payload }` } );
+		return apiFetch( { path: `/wc/v3/products/${ product }/variations${ payload }` } );
 	},
 	isDebounced: true,
 	getOptionKeywords( variation ) {

--- a/packages/date/src/index.js
+++ b/packages/date/src/index.js
@@ -7,10 +7,11 @@ import { find } from 'lodash';
 import { __ } from '@wordpress/i18n';
 import { getSettings, format as formatDate } from '@wordpress/date';
 
-/**
- * Internal dependencies
- */
-import { QUERY_DEFAULTS } from 'store/constants';
+const QUERY_DEFAULTS = {
+	pageSize: 25,
+	period: 'month',
+	compare: 'previous_year',
+};
 
 export const isoDateFormat = 'YYYY-MM-DD';
 

